docs/source/start-software.rst

```diff
@@ -140,8 +160,8 @@ Proceed with installing OpenDCS.
 If nothing is returned or the version is older then 1.8, then install
 the 1.8 or up to java 17 from https://adoptium.net/temurin/releases/ .
```

Comment:
One of the tweaks I made earlier in the week made me remember the installer won't work in Java 17. OpenDCS itself will, but the installer depends on something that was removed in 17.